XenAdmin/Dialogs/AddVGPUDialog.cs

```diff
@@ -89,6 +89,9 @@ private void PopulateComboBox()
 
                     foreach (var vgpuType in commonTypes)
                         comboBoxTypes.Items.Add(new GpuTuple(gpu_group, vgpuType, disabledTypes.ToArray())); // GPU_type item
+
+                    if (commonTypes.Count == 1 && !disabledTypes.Contains(commonTypes[0]))
+                        comboBoxTypes.SelectedItem = comboBoxTypes.Items[1]; // Items[0] is group

```

Comment:
A bug was here. Item indexed 1 doesn't exist when driver hasn't been installed on the host.